src/App.tsx

```diff
@@ -1,18 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const ItemPerPage = [3, 5, 10, 20];
```

Comment:
constants are usually declared this way
```suggestion
  const ITEMS_PER_PAGE = [3, 5, 10, 20];
```

Reply:
Done!